### PR TITLE
classes and fibonachi es6 continued

### PR DESCRIPTION
**Pull request by endrushilov**

### CODE REVIEW

File: homeworks/es6-continued/endrushilov/cart_classes.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                 count_this_elem++
+             }
+         }
+         if(count_this_elem == count){
+             console.log("return");
+            return
+         }
+         if(count_this_elem < count){
+             count_this_elem = count -  count_this_elem;
+             var new_elements = [].fill.call({length: count_this_elem}, Object.assign(this_elem));
```
Comment (komarnitskyi) [inline homeworks/es6-continued/endrushilov/cart_classes.js]: You shouldn't declare new variables inside an `if` statement

File: homeworks/es6-continued/endrushilov/cart_classes.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
+// Don't modify
+function Cart () {}
```
Comment (komarnitskyi) [inline homeworks/es6-continued/endrushilov/cart_classes.js]: `Cart` should be declared using classes syntax

File: homeworks/es6-continued/endrushilov/cart_classes.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    }    
+
+};
+class UserCart extends Item{
+    constructor(){
+        super()
+    }
+    amount(){
+        var amount = 0; 
+        for(var i = 0; i < this.goods.length; i++){
```
Comment (komarnitskyi) [inline homeworks/es6-continued/endrushilov/cart_classes.js]: Please, try to do this calculation using `reduce`

**Review by komarnitskyi — CHANGES_REQUESTED** [review]

**Comment by denysdovhan** [comment]
@komarnitskyi please, check again

**Pull request merged**